models/database.go

```diff
@@ -655,7 +655,7 @@ var databaseSchema = [][]string{
 			ADD COLUMN database_name VARCHAR NOT NULL DEFAULT ''`,
 	},
 	52: {
-		`UPDATE services SET database_name = 'postgresql' 
+		`UPDATE services SET database_name = 'postgres' 
```

Comment:
added after approvals.

Reply:
@BupycHuk do you think I can fix it by updating existing migration. ok it is better to add new migration that fixes that? e.g.

```
	53: {
		`UPDATE services SET database_name = 'postgres' 
			WHERE service_type = 'postgresql' and database_name = 'postgresql'`,
	},
```